pkg/operator/ceph/cluster/mon/mon.go

```diff
@@ -175,6 +176,9 @@ func New(context *clusterd.Context, namespace string, spec cephv1.ClusterSpec, o
 			Schedule: map[string]*MonScheduleInfo{},
 		},
 		ownerInfo: ownerInfo,
+		ClusterInfo: &cephclient.ClusterInfo{
```

Comment:
I'm not sure about this, clusterInfo is generated elsewhere and I believe this will be overridden. I need to check the code.

Reply:
I added context at this stage as the `TestReconcileMonPDB` was failing.
```
FAIL: TestReconcileMonPDB (0.00s)
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0xe0 pc=0x16aa9a0]
```

Reply:
Just looked at the code, this should not be a problem.